torchsynth/module.py

```diff
@@ -620,13 +620,18 @@ class Noise(SynthModule):
     these in a synth then choose different seeds for each instance.
     """
 
+    # Do we really want deterministic noise within each batch?
+    # https://github.com/turian/torchsynth/issues/250
```

Comment:
It's a good question -- it won't be perceptual

Reply:
But I can also see the argument for

Reply:
leave the comment for now